spec/client_spec.rb

```diff
@@ -308,11 +346,15 @@ def redis.with
           # This time we do not pass it through to Redis, in order to simulate a passing
           # call to LOAD SCRIPT followed by another NOSCRIPT error. Imagine someone
           # repeatedly calling SCRIPT FLUSH on our Redis instance.
-          expect(@manipulated_instance).to receive(:load_scripts)
+          expect(@manipulated_instance).to receive(:load_scripts).exactly(8).times
```

Comment:
"exactly 8 times" sounds very odd 🤔 maybe `at_least(1).times` would suffice? we can leave it for now, let's project authors decide :)